src/qualified_name_is_fully_qualified.hack

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+/**
+ * @reviewer I'd like to put this function on QualifiedName.
+ *           Something like QualifiedName->isFullyQualified().
+ *           The file is a SignedSource file.
```

Comment:
https://github.com/hhvm/hhast/blob/9f586c80055f73c17b95742ec97627c1d11a1deb/src/__Private/codegen/CodegenSyntax.hack#L690

Regen, then add manual subclass, e.g. https://github.com/hhvm/hhast/blob/master/src/nodes/ParameterDeclaration.hack

Make sure that you regen with the same version of HHVM/hh_parse that was last used for codegen - re-running the codegen should do nothing other than rename the base class and make it non-final

Reply:
I'll have to postpone this QoL change, because DontUseAsioJoinLinter uses the same code...